### ISSUE TITLE
Etableringsersättning

### ISSUE THREAD

**Issue by Hanifb** [Hanifb/marginal-calc#3]
Skapa ny modul för Etableringsersättning

**Comment by ohlsont** [comment]
Är [denna information](http://www.forsakringskassan.se/wps/wcm/connect/65cfd0e0-89d3-4589-bf08-7be0f0a942ef/4143_Etableringsersattning_for_ny+i_Sverige.pdf?MOD=AJPERES) korrekt och tillräcklig för issuen?

**Comment by Hanifb** [comment]
Ja, det räcker tills vidare, bra hittat! Modulen verkar vara beroende av houseHold och av bostadsBidrag. Därtill bör vi addera en underhållstödsmodul.

**Comment by ohlsont** [comment]
Ska försöka översätta dokumentet till kod, men kommer ev. kräva mera input om användarna om det ska följas korrekt.

**Comment by Hanifb** [comment]
Hmm, Om man använder.
    Person.setData("inetableringsprg", true); 
    Person.setData("etableringsprg_amount", 25); // 25% 

så borde den ha det den behöver va?

**Comment by Hanifb** [comment]
Jag börjar koda lite på detta. Lägger upp en feature fork som vi kan jobba mot. Jag är inte så van att jobba gentemot github så du får säga till om jag collaborerar fel. @hinken
